src/Libraries/GeometryUI/ImportWithUnits.cs

```diff
@@ -2,90 +2,138 @@
 using Newtonsoft.Json;
 using ProtoCore.AST.AssociativeAST;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace GeometryUI
 {
     //TODO finish the descriptions etc with resx.
 
     [NodeCategory(BuiltinNodeCategories.GEOMETRY)]
-    [NodeName("Geometry.ImportFromSatWithUnits")]
+    [NodeName("Geometry.ImportFromSATWithUnits")]
     [InPortTypes(new string[] { "var", "DynamoUnits.Unit" })]
-    //[NodeDescription("ExportToSATDescripiton", typeof(GeometryUI.Properties.Resources))]
-    //[NodeSearchTags("ExportWithUnitsSearchTags", typeof(GeometryUI.Properties.Resources))]
+    [InPortNames(new[] { "file|filePath","dynamoUnit" })]
```

Comment:
So there's just one nodemodel node for both file path and file object?

Reply:
yep, the correct overload is called at runtime, tested this a bunch and it seems solid.